dali/ft/filecopy.cpp

```diff
@@ -2685,6 +2686,9 @@ void FileSprayer::setTarget(IDistributedFile * target)
             srcFormat.separate.set("\\,");
 
         tgtFormat.set(srcFormat);
+        if ((operation == dfu_import) && ( !options->getPropBool("@keepSourceEncoding")))
```

Comment:
trivial: extra space between ( and !

Reply:
>( !options->getPropBool("@keepSourceEncoding"))

should this be :  (!options->getPropBool("@keepSourceEncoding", true)) , as in other places?

Reply:
Changed